src/main/java/uk/ac/ebi/ena/webin/cli/service/SampleService.java

```diff
@@ -139,38 +139,87 @@ protected SampleService(Builder builder) {
    * will be made to retrieve it from ENA.
    */
   public Sample getSample(String sampleId) {
+    final List<String> sampleRetrievalMessages = new ArrayList<>();
```

Comment:
From what I understand, if we do not collect these messages then we will be left with logic that is pretty similar to how it was before. Is it important to do this? As far as the user is concerned, these messages will not matter to them. i think its better to keep them unaware of where the sample came from.